lib/BaselineAnalyzer.php

```diff
@@ -24,30 +24,14 @@ public function analyze(): AnalyzerResult
          * @var BaselineError $baselineError
          */
         foreach ($this->baseline->getIgnoreErrors() as $baselineError) {
-            $errorMessage = $baselineError->message;
-
-            if (str_contains($errorMessage, ' deprecated class ') || str_contains($errorMessage, ' deprecated method ')) {
```

Comment:
IMO we could extract this if conditions into a `countDeprecation($errorMessage)`, but let the method here still work on the result of said method and adjust `$result->deprecations` here at the call-site.

Reply:
Are you thinking about this?
```php
$result->deprecations += $this->countDeprecation();
$result->classesComplexity += $this->countClassesComplexity();
...
```
Removed `$errorMessage` as it comes from a class property.

Reply:
Exactly